exercises/leap/canonical-data.json

```diff
@@ -1,39 +1,29 @@
 {
    "cases": [
       {
-         "description": "leap year in twentieth century",
-         "input": 1996,
-         "expected": true
-      },
-      {
-         "description": "odd standard year in twentieth century",
-         "input": 1997,
-         "expected": false
-      },
-      {
-         "description": "even standard year in twentieth century",
-         "input": 1998,
+         "description": "year not divisible by 4: standard year",
+         "input": 2015,
          "expected": false
       },
       {
-         "description": "standard year in nineteenth century",
-         "input": 1900,
-         "expected": false
+         "description": "year divisible by 4, not divisible by 100: leap year",
+         "input": 2016,
+         "expected": true
       },
       {
-         "description": "standard year in eighteenth century",
-         "input": 1800,
+         "description": "year divisible by 100, not divisible by 400: standard year",
+         "input": 2100,
          "expected": false
       },
       {
-         "description": "leap year twenty four hundred",
-         "input": 2400,
+         "description": "year divisible by 400: leap year",
+         "input": 2000,
          "expected": true
       },
       {
-         "description": "leap year two thousand",
-         "input": 2000,
-         "expected": true
+         "description": "year before validity of Gregorian Calendar (no leap years at all): standard year",
```

Comment:
Do I understand correctly that you would omit the boundary check completely, although there were no leap years before 1582 - thus only relying on the three rules?